blocks/autocompleters/index.js

```diff
@@ -107,8 +107,12 @@ export function blockAutocompleter( { onReplace } ) {
  * @return {Completer} Completer object used by the Autocomplete component.
  */
 export function userAutocompleter() {
-	const getOptions = () => {
-		return wp.apiRequest( { path: '/wp/v2/users' } ).then( ( users ) => {
+	const getOptions = ( search ) => {
+		let payload = '';
+		if ( search ) {
+			payload = '?search=' + search;
```

Comment:
Does this need to be [encoded](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent)?

Reply:
I assume you mean just the search part of the payload, for security? otherwise I wouldn’t think its needed, as we are searching usernames?

Reply:
ah, right '&' could cause unexpected results, good catch